src/cdi_CPEloss/Analytic_Spitzer_Eloss_Model.hh

```diff
@@ -0,0 +1,48 @@
+//----------------------------------*-C++-*-----------------------------------//
```

Comment:
Whoo hoo!! Now we have OPTIONS! :D

Reply:
One thing to maybe worry about is that by adding options I changed the analytic interface such that it now takes in `target` and `projectile` `CParticle` objects. I *think* this is a good way to handle stopping powers going forward, e.g. we get dE/dt separately for each isotope in a target material, but I don't know that the KP alpha model (fitting formula for alphas moving through a combined DT gas) can be easily reconciled to this. So maybe KP alpha should come out at some point? Maybe it is still useful to have, I don't know.

Reply:
That's a good point. I think KP Alpha can definitely come out down the road -- it was just the dumbest, most bare-bones first option I found in Edward's write-up. Once we've used it for preliminary comparisons, I'm happy to ditch it :)